src/ppom-storage.test.ts

```diff
@@ -0,0 +1,287 @@
+import { StorageBackend, PPOMStorage, StorageKey } from './ppom-storage';
+
+Object.defineProperty(globalThis, 'crypto', {
+  value: {
+    subtle: {
+      digest: () => new ArrayBuffer(12),
+    },
+  },
+});
+
+const checksum =
+  '409a7f83ac6b31dc8c77e3ec18038f209bd2f545e0f4177c2e2381aa4e067b49';
+class MockStorageBackend implements StorageBackend {
+  data: undefined;
+
+  constructor(data?: any) {
+    this.data = data;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
If an argument is not used, it can be prepended with `_` to tell ESLint that it is intentional. That should remove the need to add these ignore lines.

Reply:
Done, thanks